thavalon-server/src/lobby/lobby_impl.rs

```diff
@@ -170,14 +210,79 @@ impl Lobby {
         LobbyResponse::Standard(Ok(()))
     }
 
-    /// [WIP] - Starts a game.
+    /// Handles a change to the player list, due to a player joining or leaving the game.
+    async fn on_player_list_change(&mut self) {
+        // Only broadcast to players if the game hasn't started yet.
+        // TODO: Maybe a helpful message to players that someone has disconnected.
```

Comment:
Players will get known by the list updating right? Or do you mean a more detailed message on who disconnected?

Reply:
I mean during the game. Like if a player disconnects during the mission phase, it might be nice to tell other players. Not a requirement though.